pages/_app.tsx

```diff
@@ -1,8 +1,14 @@
 import '../styles/globals.css'
 import type { AppProps } from 'next/app'
+import { store } from '../store'
+import { Provider } from 'react-redux'
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />
+  return (
```

Comment:
This probably won't work since you need redux next wrapper to be able to use store inside next.js application